Rename Hover Image Preview to Image Preview

alvesjtiago opened the pull request:
<!--
Your pull request will be reviewed automatically and by a human.

The manual review may take several days or weeks, depending on the reviewer's availability and workload.
If you haven't received a comment on your pull request and it wasn't merged either,
it just hasn't been reviewed yet.

---

Please ensure the automated reviews pass. 
Follow the instructions provided, if necessary.
You can speed up the process
by [running some tests locally](https://packagecontrol.io/docs/submitting_a_package#Step_7).

You can trigger @packagecontrol-bot to re-evaluate your pull request
by pushing a commit or closing and reopening your pull request.
Do **NOT** open a new pull request!

In general, make sure you:

 1. Used `"tags": true` and not `"branch": "master"` 
    (versioning docs: <https://packagecontrol.io/docs/submitting_a_package#Step_4>)
 2. Added a README to your repository so that users (and reviewers) 
    can understand what your package provides.
 
You should proceed with a short description of what the package does
and, in case one or multiple similar package already exists, 
why you believe it is different and needed
below this line. -->

This PR renames the Hover Image Preview package to Image Preview in order to make it easier to discover.

FichteFoll commented:
I personally find the old name much more descriptive of what it does and having more search names will hardly make it less discoverable when there isn't any competition. Imagine searching for "image hover", which [currently works](https://packagecontrol.io/search/image%20hover).

Either way, tests are failing.

alvesjtiago commented:
@FichteFoll the reasoning behind it is that the plugin now works without necessarily hovering.  I've just fixed the ordering issue and added a label called hover. Will this provide the same output as https://packagecontrol.io/search/image%20hover? Thank you

FichteFoll commented:
No, labels are not covered in the search terms, but the description is, so you might want to add some terms there. If you didn't know, the description is sourced from your repo description (but you can also supply it directly here).
